.github/workflows/php.yml

```
@@ -0,0 +1,61 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Is this intentionally limited to the `master` branch? I think the configuration I usually have is just

```suggestion
  push:
  pull_request:
```

with no configuration for either trigger.